### ISSUE TITLE
Broken `DeleteExecutionsWithWorkflows` migration

### ISSUE THREAD

**Issue by terenaa** [n8n-io/n8n#5183]
**Describe the bug**
Migration collects **workflow execution** IDs instead of **workflow** IDs preparing to create FK.

```ts
const workflowIds: Array<{ id: number }> = await queryRunner.query(`
  SELECT id FROM \`${tablePrefix}execution_entity\`
`);

await queryRunner.query(
  `DELETE FROM \`${tablePrefix}execution_entity\`
  WHERE workflowId IS NOT NULL
  ${workflowIds.length ? `AND workflowId NOT IN (${workflowIds.map(({ id }) => id).join()})` : ''}`,
);
```

Field `id` from `execution_entity` has no relation with `workflowId`.

**To Reproduce**

1. Create an entry in `execution_entity` with `workflowId` which does not exist in `workflow_entity`.
2. `ALTER TABLE ...` will fail.

**Expected behavior**
Variable `workflowIds` should contain ids from `workflow_entity` table.

**Environment:**

- n8n Version 0.211.0

**Comment by Joffcom** [comment]
Hey @terenaa,

Thanks for reporting this one and a great spot, It looks like @csuermann has popped in a fix already which should make it into the next release.

**Comment by janober** [comment]
Fix got released with `n8n@0.212.0`

**Comment by ifokeev** [comment]
Still not working
